### PR TITLE
added colours to pdf graphs and fixed bar-chart

### PR DESCRIPTION
**Pull request by megansam5**
Edited the weekly email pdf to have colours in the graphs and a correct bar chart.

### CODE REVIEW

File: weekly-emailing/graphs.py [hunk 1]
```diff
@@ -7,22 +7,23 @@
...
-            range=[30, 300])),
-        tooltip=['topic_name', 'source_name',
-                 'avg_polarity_score', 'article_count']
+        x=alt.X('avg_polarity_score:Q', title="Average Polarity Score"),
+        y=alt.Y('topic_name:O', title="Topic"),
+        color=alt.Color('source_name:N', title="News Source",
+                        scale=color_scale),
+        tooltip=[alt.Tooltip(field="topic_name", title="Topic"),
+                 alt.Tooltip(field="avg_polarity_score",
+                             title="Average polarity score"),
```
Comment (YakMan101) [inline weekly-emailing/graphs.py:21]: Nice adding the cleaner title formatting.

File: weekly-emailing/graphs.py [hunk 2]
```diff
@@ -32,32 +33,35 @@ def create_sentiment_distribution_chart(df):
...
         grid=False
     ).configure_view(
         strokeWidth=0
     )
 
-    return chart_to_base64(faceted_chart)
+    return chart_to_base64(chart)
 
 
 def chart_to_base64(chart):
```
Comment (YakMan101) [inline weekly-emailing/graphs.py:67]: Just a styling consideration, even though the script would work, could you define the function before it is used in `create_bar_graph(df)`

File: weekly-emailing/graphs.py [hunk 3]
```diff
@@ -32,32 +33,35 @@ def create_sentiment_distribution_chart(df):
 
 
 def create_bar_graph(df):
```
Comment (YakMan101) [inline weekly-emailing/graphs.py:35]: Add type hints pleas e.g:

```
def create_bar_graph(df: pd.DataFrame) -> str:
```

```
def chart_to_base64(chart: alt.Chart) -> str:
```

**Review by YakMan101 — APPROVED** [review]

File: weekly-emailing/graphs.py (hunk 3 above)
Comment (megansam5) [reply weekly-emailing/graphs.py:35]: Good notice, I'll do those now!

**Pull request merged**